lib/src/data/datasource/remote/api_service.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import '../../models/genre_response.dart';
+import '../../models/movie_response.dart';
+import 'i_api_service.dart';
+
+class ApiService implements IApiService {
+  static const String _apiKey = '025aa98c350d497beb053d9b5e169637';
+  static const String _baseUrl = 'https://api.themoviedb.org';
+  static const String _popularEndpoint = '/3/movie/popular';
+  static const String _nowPlayingEndpoint = '/3/movie/now_playing';
+  static const String _topRatedEndpoint = '/3/movie/top_rated';
+  static const String _upcomingEndpoint = '/3/movie/upcoming';
+  static const String _genresEndpoint = '/3/genre/movie/list';
+  static const String moviesException = 'Failed to load movies';
+  static const String genresException = 'Failed to load genres';
+
+  const ApiService();
+
+  @override
+  Future<MovieResponse> getPopular() async {
+    final response = await http
+        .get(Uri.parse(_baseUrl + _popularEndpoint + '?api_key=' + _apiKey));
```

Comment:
save this '**?api_key='** into a class constant

Reply:
done